Alvaro Gaming/PythonGaming_p3.py

```diff
@@ -14,79 +14,114 @@
 BLUE = (0, 0, 255)  # Blue
 GREEN = (0, 255, 0)  # Green
 
-
 # assigning FPS Value
 FPS = 60
 FramePerSec = pygame.time.Clock()
 
 # Setting up other variables that will be used in program
-SCREEN_WIDTH = 400
+SCREEN_WIDTH = 800
 SCREEN_HEIGHT = 600
 SPEED = 5
-SCORE = 0
+SCORE_1 = 0
+SCORE_2 = 0
 
-#Setting up Fonts
-font = pygame.font.SysFont("Verdana",60)
-font_small = pygame.font.SysFont("Verdana",20)
+# Setting up Fonts
+font = pygame.font.SysFont("Verdana", 60)
+font_small = pygame.font.SysFont("Verdana", 20)
 game_over = font.render("Game_Over", True, BLACK)
 
-background = pygame.image.load("AnimatedStreet.png")
+background = pygame.image.load("space.png")
 
 # Create white screen
 DISPLAYSURF = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
 DISPLAYSURF.fill(WHITE)
-pygame.display.set_caption("Alvy Game")
+pygame.display.set_caption("Alvy Game: Pong")
+
 
 class Enemy(pygame.sprite.Sprite):
```

Comment:
Since this is actually a "ball" that's a lot more fitting of a name. At first I thought this was for the second paddle.